textract2page/convert_aws.py

```diff
@@ -712,7 +712,32 @@ def convert_file(json_path: str, img_path: str, out_path: str) -> None:
 
     Arguments:
         json_path (str): path to input JSON file
-        img_path (str): path to input JPEG file
+        img_path (str): path to input image file
+        out_path (str): path to output XML file
+    """
+    
+    # get absolute image coordinates
+    pil_img = Image.open(img_path)
+    img_width = pil_img.width
+    img_height = pil_img.height
+    pil_img.close()
+    
+    convert_file_without_image(json_path, img_path, img_width, img_height, out_path)
+
+
+def convert_file_without_image(json_path: str, img_path: str, img_width: int, img_height: int, out_path: str) -> None:
```

Comment:
Note: this function should also be exported (i.e. added to the list in `__init__.py`'s `__all__`)